### PR TITLE
help subcommand

### PR DESCRIPTION
**Pull request by RafaelGomides**
Use `$ go run main.go -help`

### CODE REVIEW

File: cmd/gaze/main.go [hunk 1]
```diff
@@ -18,21 +18,53 @@ const (
...
-	return &argument{height: uint16(*height), width: uint16(*width)}
+	return &argument{height: uint16(*height), width: uint16(*width), help: bool(*help)}
 }
 
 func main() {
 	arg := parseArguments()
-	k := algos.NewPrim(arg.height, arg.width)
-	k.Generate()
-	k.Board.Write(os.Stdout)
+	if arg.help == true {
```
Comment (wliao008) [inline cmd/gaze/main.go]: minor thing, given `help` is of bool
```
if arg.help {
...
}
```
would be enough

Comment (wliao008) [inline cmd/gaze/main.go]: minor thing, given `help` is of bool
```
if arg.help {
...
}
```
would be enough

File: cmd/gaze/main.go [hunk 2]
```diff
@@ -18,21 +18,53 @@ const (
...
+		fmt.Println(helpReturn())
```
Comment (wliao008) [inline cmd/gaze/main.go]: we could get rid of the `else` if we exit here with `os.Exit(0)`

File: cmd/gaze/main.go [hunk 3]
```diff
@@ -18,21 +18,53 @@ const (
...
+	return `
+	Use params to improve your experience.
+	- To set Heigth: -h <SIZE_IN_INTEGER>
+	- To set Weigth: -w <SIZE_IN_INTEGER>
+
+	Example: 
+
+	$ go run main.go -h 12 -w 12
+
+	_ _ _ _ _ _ _ _ _ _ _
```
Comment (wliao008) [inline cmd/gaze/main.go]: some space format issue seems like.

Comment (RafaelGomides) [reply cmd/gaze/main.go]: Updated.

File: cmd/gaze/main.go (hunk 2 above)
Comment (RafaelGomides) [reply cmd/gaze/main.go]: Done

**Pull request merged**